dataquality/loggers/data_logger/image_classification.py

```diff
@@ -137,7 +138,13 @@ def _prepare_content(
                 export_format=export_format,
             )
             df = vaex.open(f"{temp_dir}/*.arrow")
-        dataset["text"] = df["object_path"].to_numpy()
+        df = df.to_pandas_df()
+        # df has columns "file_path", "object_path" we merge with original dataset
+        # on imgs_location_colname and rename "object_path" to "text"
+        dataset = dataset.merge(df, left_on=imgs_location_colname, right_on="file_path")
+        dataset["text"] = dataset["object_path"]
+        for f in map(os.remove, glob.glob(f"{temp_dir}/*.arrow")):
+            pass
```

Comment:
what is this doing?

Reply:
map is a generator

Reply:
can we make this more clear? 
```suggestion
        for f in glob.glob(f"{temp_dir}/*.arrow"):
            os.remove(f)
```